frontend/src/components/Account/common/ProfileDetails/ProfileDetails.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+import "./ProfileDetails.scss";
+
+const renderField = (field = {}) => {
+  let processedValue = field.value;
+  if (typeof field.value === "boolean") {
+    processedValue = field.value ? "Da" : "Nu";
+  }
```

Comment:
The reason for the if statement for the Boolean check is because this function accepts other types besides Boolean, and only when the value is Boolean would it make sense to set it to 'Da'/'Nu' .

Reply:
Makes sense 👍